### PR TITLE
feat : Prouducts page

### PR DESCRIPTION
**Pull request by shuklahimanshu81**
Products page contains almost all the products . User can also sort and filter the products base on different filters.
and also using to add to wishlist button user can add items to wishlist and also to cart using add to cart buttons.

### CODE REVIEW

File: src/Components/Filters/Filters.jsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
+import { FaStar } from 'react-icons/fa';
+import './Filters.css'
+import { useProduct } from '../../Context/filterContext';
+
+const Filters = () => {
+   const {dispatch} = useProduct();
+
+
```
Comment (Jia-A) [inline src/Components/Filters/Filters.jsx:8]: Remove this extra space maybe.

File: src/Components/Products/Products.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+const Products = () => {
+    
+ const { finalProduct} = useProduct()
+ const {cartDispatch} = useCart();
+const {wishlistDispatch} = useWishlist()
+console.log(finalProduct)
+
+return <div className="products-page">
+  <Filters />
+<section className="products-section">
```
Comment (Jia-A) [inline src/Components/Products/Products.jsx:19]: Nice use of semantic HTML.

File: src/Context/cartContext.jsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+
+import {createContext, useContext, useReducer} from 'react'
+import { useProduct } from './filterContext';
+
+const cartContext = createContext();
+
+const useCart = () => useContext(cartContext)
+
+
```
Comment (Jia-A) [inline src/Context/cartContext.jsx:9]: remove extra spacing.

**Review by Jia-A — APPROVED** [review]
Suggestions : Just remove extra spacing from some places. 
Everything looks good to me, nice naming conventions and folder structure. 
You can merge it.

**Review by srishti-maurya — APPROVED** [review]
looks good

**Pull request merged**